src/components/Utils/orderResource.js

```diff
@@ -29,10 +29,10 @@ export const updateOrderResource = (order, mutator, changedProps) => {
 
 export const createOrderResource = (order, mutator) => {
   const clonedOrder = cloneDeep(order);
-  const { numberPrefix = '', numberSuffix = '', po_number: orderNumber = '' } = clonedOrder;
+  const { numberPrefix = '', numberSuffix = '', poNumber: orderNumber = '' } = clonedOrder;
```

Comment:
in case of `poNumber` there is no need in `orderNumber`, it could be just `poNumber = ''`